docs/learn/advanced/05-encoding.md

```diff
@@ -34,14 +34,14 @@ tree.
 
 For storage encoding, module developers are encouraged to use Protobuf encoding for their types
 but may choose any encoding schema they like. The
-[collections](../../build/packages/02-collections.md) package automatically handles encoding and
+[collections](https://github.com/cosmos/cosmos-sdk/blob/collections/v0.4.0/collections/README.md) package automatically handles encoding and
```

Comment:
when the docs build script this gets fixed, it can be reverted.

Reply:
is it broken?

Reply:
it should work and resolve this

Reply:
yes